### PR TITLE
A few small grammar updates

### PR DESCRIPTION
**Pull request by BBlackwo**

### CODE REVIEW

File: i18n/md/collections/2020/en/categories/all.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ language: en
 
 2020 is a very special year for many reasons. The most important one: for the first time in 5 years the overall winner of the _Rising Stars_ is not called {vuejs} but {deno}... what a surprise!
 
-#### The Year of the Deno
+#### The Year of Deno
```
Comment (BBlackwo) [inline i18n/md/collections/2020/en/categories/all.md:8]: I guess you might not even need this heading as you moved the Tailwind content to the new category

Comment (michaelrambeau) [reply i18n/md/collections/2020/en/categories/all.md:8]: Yes, we don't need it any more.
I wanted to make a joke after having listened to a good song called "The Year of the Cat" (from the seventies!) actually 😅

**Comment by michaelrambeau** [comment]
👍 Again, thank you very much Benjamin for your careful reviews!

File: i18n/md/collections/2020/en/categories/all.md (hunk 1 above)
Comment (BBlackwo) [reply i18n/md/collections/2020/en/categories/all.md:8]: Gotcha, haha. I guess I didn't get the joke 😄

**Pull request merged**